MCSync/frmConfig.cs

```diff
@@ -137,5 +132,135 @@ private void cmdBrowseOResources_Click(object sender, EventArgs e)
 
             txtOResourcesDirectory.Text = browser.SelectedPath;
         }
+
+        private void cmdMinecraftFolder_Click(object sender, EventArgs e)
+        {
+            BetterFolderBrowser browser = new BetterFolderBrowser();
+            browser.Title = "Select .minecraft folder";
+            browser.Multiselect = false;
+            browser.RootFolder = txtMinecraftFolder.Text;
+            if (browser.ShowDialog() != DialogResult.OK)
+            {
+                return;
+            }
+
+            txtMinecraftFolder.Text = browser.SelectedPath;
+        }
+
+        private void overrideFoldersCheckBox_CheckedChanged(object sender, EventArgs e)
+        {
+            txtModsDirectory.Enabled = overrideFoldersCheckBox.Checked;
+            txtResourcePacksDirectory.Enabled = overrideFoldersCheckBox.Checked;
+            txtConfigDirectory.Enabled = overrideFoldersCheckBox.Checked;
+            txtOResourcesDirectory.Enabled = overrideFoldersCheckBox.Checked;
+            cmdBrowseMods.Enabled = overrideFoldersCheckBox.Checked;
+            cmdBrowseResourcePacks.Enabled = overrideFoldersCheckBox.Checked;
+            cmdBrowseConfig.Enabled = overrideFoldersCheckBox.Checked;
+            cmdBrowseOResources.Enabled = overrideFoldersCheckBox.Checked;
+
+            txtMinecraftFolder.Enabled = !overrideFoldersCheckBox.Checked;
+            cmdMinecraftFolder.Enabled = !overrideFoldersCheckBox.Checked;
+
+            if(!overrideFoldersCheckBox.Checked)
+            {
+                txtModsDirectory.Text = txtMinecraftFolder.Text + "\\mods";
+                
+                if(rbServer.Checked)
+                {
+                    txtResourcePacksDirectory.Text = txtMinecraftFolder.Text + "\\config\\immersiverailroading";
+                }
+                else
+                {
+                    txtResourcePacksDirectory.Text = txtMinecraftFolder.Text + "\\resourcepacks";
+                }
+
+                txtConfigDirectory.Text = txtMinecraftFolder.Text + "\\config";
+                txtOResourcesDirectory.Text = txtMinecraftFolder.Text + "\\oresources";
+            }
+        }
+
+        private void rbServer_CheckedChanged(object sender, EventArgs e)
+        {
+            if(rbServer.Checked)
```

Comment:
We should probably only do this logic if `overrideFoldersCheckBox` isn't checked, much like how you did in the method above

Reply:
Fixed